recipe/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "azureml_mlflow" %}
+{% set version = "1.55.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/py3/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py3-none-any.whl
+  sha256: 048cd60c483d778b1f52fc219a32c925706dceac871d829946099609959c0eef
+
+build:
+  number: 0
+  # skipping because of missing mlflow-skinny for s390x
+  skip: True  # [s390x]
+
+requirements:
+  host:
+    # patch tools are in host section since they are necessary only during execution of build script
+    - patch     # [not win]
+    - m2-patch  # [win]
+    - python
+    - pip
+    - wheel
+  run:
+    - python
+    - six
+    - cryptography
+    - mlflow-skinny
+    - azure-identity
+    - azure-common
+    - azure-core >=1.8.0,<2.0.0,!=1.22.0
+    - azure-mgmt-core >=1.2.0,<2.0.0
+    - azure-storage-blob >=12.5.0,<=12.19.0
+    - python-dateutil >=2.7.3,<3.0.0
+    - msrest >=0.6.18
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
+  imports:
+    - azureml
+    - azureml.mlflow
+    - azure.core
+    - azure.mgmt.core
+    - azure.storage.blob
+    - mlflow
+
+about:
+  home: https://pypi.org/project/azureml-mlflow
+  license: MOSA
```

Comment:
`LicenseRef-Microsft-EULA` would probably be more accurate. `license` must be an SPDX supported license. (https://spdx.github.io/spdx-spec/v2-draft/SPDX-license-expressions/)

Reply:
Thank you!